### PR TITLE
[fix bug 1404542] Adds a send to device snippet for AS.

### PR DESCRIPTION
**Pull request by jpetto**
@glogiotatidis - Along with general review, there's a question for you (commented in code) regarding the availability/use of `{{ snippet_id }}`.

### CODE REVIEW

File: activity-stream/send-to-device.html [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+  
+        function sendEmailToBasket() {
+          const formData = new FormData();
+          formData.append('email', input.value);
+          formData.append('newsletters', '{{ message_id_email }}');
+          // REVIEWER (Giorgos, most likely :)
+          // In a previous send to device snippet (/campaings/mobile/mobile-snipped-widget.html),
+          // the URL below has "{{ snippet_id }}" appended to the end. Is this value still available?
+          // It appears to be empty testing locally in simple_template_server, but maybe it's available
+          // in the real application?
```
Comment (glogiotatidis) [inline activity-stream/send-to-device.html]: `{{ sinppet_id }}` works on both simple_template_server and the real application, just tested it. Note that `snippet_id` on the dev servers starts with `0` so maybe you mistook it for url encoding?

File: activity-stream/send-to-device.html [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+          formData.append('source_url', encodeURIComponent('https://snippets.mozilla.com/'));
+  
+          // Maybe change to the following?
+          // formData.append('source_url', encodeURIComponent('https://snippets.mozilla.com/{{ snippet_id }}'));
```
Comment (glogiotatidis) [inline activity-stream/send-to-device.html]: I suggest `https://snippets.mozilla.com/show/{{ snippet_id }}` which displays a preview of the snippet. (if snippet is public)

File: activity-stream/send-to-device.html (hunk 1 above)
Comment (jpetto) [reply activity-stream/send-to-device.html]: Totally lost it in the URL encoding. 😁

**Comment by jpetto** [comment]
@glogiotatidis Updated. Thanks!

**Comment by glogiotatidis** [comment]
Thanks!

**Comment by glogiotatidis** [comment]
Both pushed to production

**Pull request merged**